### PR TITLE
added commenting functionality - write_comment

### PR DESCRIPTION
**Pull request by hack-r**
Tested comment functionality successfully

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +51,7 @@
     "test",
     "docs"
   ],
-  "black": {
+  "all": {
```
Comment (castle-lemongrab) [inline package.json:54]: No thanks.

**Review by castle-lemongrab — CHANGES_REQUESTED** [review]
Do not edit portions of the code that are not necessary to implement your feature. Please do not check in large binaries. Please do not use others' repositories to make your political statements. Rejected; we'll implement this ourselves.

**Comment by hack-r** [comment]
Yea, fuck you too. I’ll add what I want, take it or leave it ungrateful asshole.

**Comment by hack-r** [comment]
PS Since you want to take my idea while insulting and rejecting my contributions, I’ll be rebranding and publishing my version. So long, and thanks for all the fish.

ALL lives matter. Support your local police!! Donald Trump is the greatest President in American history!!!

**Comment by castle-lemongrab** [comment]
The license permits you to do this, and also to check large binaries into your own repository. Have fun!

**Pull request closed without merging**